includes/components/class-attribute.php

```diff
@@ -400,6 +469,7 @@ public function register_attributes() {
 						'searchable'     => (bool) $attribute_object->hp_searchable,
 						'filterable'     => (bool) $attribute_object->hp_filterable,
 						'sortable'       => (bool) $attribute_object->hp_sortable,
+						'unique'         => (bool) $attribute_object->hp_unique,
```

Comment:
Please let me know in short how the solution works (in tg is ok), and why we use the "unique" flag and naming here, the task is related to "readonly" functionality and if possible we should re-use the "readonly" parameter of the field class (and, for example, add a setting for it in the field init)

Reply:
Sent description in TG